specs/waku/v2/waku-v2-rpc-api.md

```diff
@@ -117,6 +117,8 @@ The following structured types are defined for use on the Relay API:
 | `payload` | `String` | mandatory | The payload being relayed |
 | `contentTopic` | `Number` | optional | Message content topic for optional content-based filtering |
 
+ > **_NOTE:_** `WakuRelayMessage` maps directly to a [`WakuMessage`](#WakuMessage), except that the latter contains an explicit message `version`. For `WakuRelay` purposes, the versioning is handled by the API.
```

Comment:
Could you please explain more about this part? what does mapping mean?

Reply:
For now it just means that, for now, WakuMessage.payload == WakuRelayMessage.payload and WakuMessage.contentTopic == WakuRelayMessage.contentTopic. The only difference then is that `WakuMessage` contains a `version` field which is confusing to have on the non-Nim API, since the `version` indicates payload encryption which will eventually be handled by the API. Hope that makes it a bit clearer? :)